substrate/frame/contracts/src/tests.rs

```diff
@@ -457,28 +436,21 @@ parameter_types! {
 	pub static UnstableInterface: bool = true;
 }
 
+#[derive_impl(crate::config_preludes::TestDefaultConfig as crate::DefaultConfig)]
 impl Config for Test {
```

Comment:
Some of the default trait are overridden here as we needs to use non const values to test different scenarios